docs/csharp/whats-new/csharp-6.md

```diff
@@ -508,12 +508,12 @@ if any, is lost.
 ## Index Initializers
 
 *Index Initializers* is one of two features that make collection
-initializers more consistent. In earlier releases of C#, you could use
-*collection initializers* only with sequence style collections:
+initializers more consistent with index usage. In earlier releases of C#, you could use
+*collection initializers* only with sequence style collections, including <xref:System.Collections.Generic.Dictionary%602> with braces around Key-Vlue pairs:
```

Comment:
There is a typo and what about rephrasing: *...around Key-Vlue pairs* -> *...around key and value pairs*

Reply:
@DevonAleshire thank you very much for updating the text.